### DISCUSSION TITLE
Add Validation Set to RF Optimizations Metrics

### DISCUSSION THREAD

**Discussion by lusis-ai** [google/yggdrasil-decision-forests#102] [Ideas]
Today optimization score (tuner) is based on OOB from the training set.

In many applications it would be more interesting to enable adding a validation set and add this score to each optimization result. That's especially the case for some Time Series where test must only happened on future data.
